plugins/chat/lib/chat/message_creator.rb

```diff
@@ -233,7 +233,8 @@ def post_process_resolved_thread
       return if resolved_thread.blank?
 
       resolved_thread.increment_replies_count_cache
-      resolved_thread.add(@user)
+      current_user_thread_membership = resolved_thread.add(@user)
```

Comment:
This just fixes an issue where you'd still see a blue dot after sending a message in the thread until you reopened it.